src/app/modules/shared/components/details-fields/details-fields.component.html

```diff
@@ -1,21 +1,30 @@
 <form [formGroup]="entryForm" (ngSubmit)="onSubmit()">
-  <label *ngIf='canMarkEntryAsWIP'><input id='isEntryRunning' type="checkbox" (change)="onGoingToWorkOnThisChange($event)" [checked]="goingToWorkOnThis"> I am working on
-    this</label>
+  <label *ngIf="canMarkEntryAsWIP">
+    <input
+      id="isEntryRunning"
+      type="checkbox"
+      (change)="onGoingToWorkOnThisChange($event)"
+      [checked]="goingToWorkOnThis"
+    />
+    I am working on this
+  </label>
 
   <div class="form-group row">
     <label class="col-12 col-sm-2 col-form-label">Project:</label>
     <div class="col-12 col-sm-10 autocomplete">
       <ng-autocomplete
-        (selected)='onSelectedProject($event)'
-        (inputCleared)='onClearedComponent($event)'
+        (selected)="onSelectedProject($event)"
+        (inputCleared)="onClearedComponent($event)"
         formControlName="project_name"
         [data]="listProjects"
         [searchKeyword]="keyword"
         historyIdentifier="projectsSelected"
         notFoundText="No projects found"
         placeHolder="Enter the project name"
         [itemTemplate]="itemTemplate"
-        [notFoundTemplate]="notFoundTemplate">
+        [notFoundTemplate]="notFoundTemplate"
+        [class.valid]="project_name.invalid"
```

Comment:
I know I said to add a feedback. But looking at the UI, It doesn't look ok for me at first. Let's remove this line `[class.valid]="project_name.invalid"` and think another way of feedback.

![image](https://user-images.githubusercontent.com/17350786/107064622-b93f4f00-67a9-11eb-9b0d-5e5ff8e58904.png)

Reply:
We can also discuss it with the team, to get a consensus.